src/pybind/mgr/dashboard/requirements-lint.txt

```diff
@@ -8,3 +8,4 @@ rstcheck==3.3.1
 autopep8==1.5.7
 pyfakefs==4.5.0
 isort==5.5.3
+jsonschema
```

Comment:
i just remember that we should be pinning the versions. else they'll probably cause issues later on.